cev_eris.dme

```diff
@@ -935,6 +935,7 @@
 #include "code\game\objects\items\stacks\medical.dm"
 #include "code\game\objects\items\stacks\nanopaste.dm"
 #include "code\game\objects\items\stacks\rods.dm"
+#include "code\game\objects\items\stacks\knife.dm"
```

Comment:
Already included.

Reply:
I have no idea why this is here.

Reply:
@Kegdo

Reply:
that's throwing knives, ye. probably got added in that pull origin stuff